src/v2/Apps/Artists/artistsRoutes.tsx

```diff
@@ -18,7 +18,7 @@ const ArtistsByLetterRoute = loadable(
   }
 )
 
-export const artistsRoutes: RouteConfig[] = [
+export const routes: RouteConfig[] = [
```

Comment:
Why are we renaming these, if we need to import as `artistsRoutes` anyways?

Reply:
It felt like a reasonable time to make the names consistent. There are potential benefits of having a consistent interface if we wanted to make the route loader smarter in the future.

I will revert them if you'd like.

Reply:
`routes` seemed to be the preference.

Reply:
Not having them named `artistsRoutes`, `artworkRoutes`, etc, is actually tech debt that we've been needing to fix because its causing us problems differentiating between different relay queries (due to the need for relay output fragments to include the file name). For example, this came up yesterday:

https://artsy.slack.com/archives/CP9P4KR35/p1607534922411300

> [RelayCache#set] Success {"queryId":"routes_ShowInfoQuery","variables":{"slug": slug}}

If we had this `route` file named `showRoutes` then there would have been no confusion here.

Reply:
Those query names are independent of the named export. As far as the export if concerned `routes` is good and appropriate from a systems architecture perspective, plus a stack trace will provide the correct data. 

💯 agree that those queries should be updated, we have similar issues with anonymous functions throughout force. Taking a look at the exports for the files I modified they suffer from the same inconsistency so let's not block changes for unrelated reasons: 

https://github.com/artsy/force/blob/master/src/v2/Apps/Conversation/routes.tsx#L16

Reply:
The thing is is that relay, from a compiler level, requires that the file name be prefixed in the query; we can't update the query name without updating the filename, relay will fail to compile.

Reply:
Sounds like everything is good as is then, I don't plan on renaming the files.